New package: SNaQ v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: SNaQ
- Repository: https://github.com/JuliaPhylo/SNaQ.jl
- Created by: @jjustison
- Version: v1.0.0
- Commit: 7b5457aed10ad5ba3aea35fdd9858176686f3268
- Reviewed by: @jjustison
- Reference: https://github.com/JuliaPhylo/SNaQ.jl/commit/7b5457aed10ad5ba3aea35fdd9858176686f3268#commitcomment-156067409
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6c032bea0a465ee7972144f6a97727d3338048bfb36e6f72659119212e1e99855b9af098fc054d734c5804013412b5b4febcc088b661501309a90e7506814537ea4895a260ca9d0b8d8530337cecab5f359636508f06a4525111dc937786a8ab0a2c2a7ec3a6182203104eb44b53c4a0d452725a3f93af692be530438e86ac7cd0623ec4170d7076534ea3329d5002d484510c3487666f1b49cfdc6f6bc156fd308eb58a53556c8d7a8a6579e3471b8d9 -->

goerz commented:
[noblock] I would probably be okay with the name, but if you can't find / convince a registry maintainer to merge it, `PhyloSNaQ` might be a good alternative.

JuliaRegistrator committed:
New package: SNaQ v1.0.0

UUID: c2bf7a07-44d2-4d11-b5e2-76dc5aa199d6
Repo: https://github.com/JuliaPhylo/SNaQ.jl.git
Tree: 56afe484fd632cb0d70b2f2553c0ec3d08b94b4b

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

jjustison commented:
Thanks for the guidance, @goerz! I'll try pinging some of the folks in the Julia slack

The core functions from this package package  were originally a part of [PhyloNetworks](https://github.com/JuliaPhylo/PhyloNetworks.jl) but we recently split things out to keep the scope more focused and have things a bit more modular. SNaQ is the name of the approach published [here](https://doi.org/10.1371/journal.pgen.1005896) and has been known as such in the phylogenetic community, so we would like to keep the name as `SNaQ.jl`

goerz commented:
Sure, I understand, but I can't override the 5-character rule. You would need a full registry maintainer to merge it manually
